app/views/development_metrics/departments.erb

```diff
@@ -1,42 +1,51 @@
-<%= render 'shared/department/sidebar'%>
-<div class="metrics-container container">
+<div class="container-fluid">
   <div class="row">
-    <div class="col-md-12">
-      <%= render 'shared/department/nav-filter' do |f|%>
+      <div class="metrics-container col-md-12">
         <div class="row">
-          <div class="col-md-5 period-input">
-            <%= render 'shared/interval-filter', form: f %>
+          <div class="col-md-12">
+            <%= render 'shared/department/nav-filter' do |f|%>
+              <div class="row">
+                <div class="col-md-5 period-input">
+                  <%= render 'shared/interval-filter', form: f %>
+                </div>
+                <div class="col-md-7">
+                </div>
+              </div>
+            <% end %>
           </div>
-          <div class="col-md-7">
+        </div>
+        <% if @review_turnaround && @merge_time %>
+          <div class="row">
+            <%= render 'development_metrics/code_climate/code_climate_department_summary',
```

Comment:
Shouldn't we render this even if there is no `@review_turnaround` and `@merge_time`?

Reply:
uhmm 🤔 i think that would be different based on the design

Reply:
Would it? If there is no RT or MT data, but there is CC data, shouldn't we show the CC data?